visualization/app/codeCharta/ui/codeMap/codeMap.component.spec.ts

```diff
@@ -85,6 +98,34 @@ describe("ColorSettingsPanelController", () => {
 		})
 	})
 
+	describe("onSharpnessModeChanged", () => {
+		beforeEach(() => codeMapController.onSharpnessModeChanged())
+
+		it("should call threeViewerService stopAnimate", () => {
+			expect(threeViewerService.stopAnimate).toHaveBeenCalled()
+		})
+
+		it("should call threeViewerService destroy", () => {
+			expect(threeViewerService.destroy).toHaveBeenCalled()
+		})
+
+		it("should call codeMapController $postLink", () => {
+			expect(codeMapController.$postLink).toHaveBeenCalled()
+		})
+
+		it("should call threeViewerService autoFitTo", () => {
+			expect(threeViewerService.autoFitTo).toHaveBeenCalled()
+		})
+
+		it("should call threeViewerService animate", () => {
+			expect(threeViewerService.animate).toHaveBeenCalled()
+		})
+
+		it("should call threeViewerService animateStats", () => {
+			expect(threeViewerService.animateStats).toHaveBeenCalled()
+		})
```

Comment:
Small idea (nit): We could do that in one test function by iterating over an array of the function names and add the expects automatically.

Reply:
sure :)